### PR TITLE
Update color palette for accessibility

### LINKED ISSUE #55: Improve color choices for accessibility
Some of the colors chosen are too similar for people with color deficiency people to differentiate.

### PR DESCRIPTION
**Pull request by kartikcho**
The current color palette is too similar for people with color deficiency people to differentiate. Made hover colors much darker in contrast and set user timing marks to grey color.

This is a re-PR that addresses comments from #113 and resolves #55 

image

### CODE REVIEW

**Comment by taneliang** [comment]
The new user timing mark colors are nice!

I like that the yellows have been reverted. The colors are also almost the same as the ones on master, which I guess is a good thing? I didn't have a problem with the blue hover colors on master, so I can't really say if this fixed any accessibility issues (I didn't run this through any color filters so I'm not sure if people with protanopia or other color deficiencies will appreciate any of the changes here).

Unfortunately, this does mean that the accessibility issues I had from master aren't fixed in this PR:

* The hovered/unhovered reds are too similar, similar enough that they don't look like they changed color when I hover over them
* (Comment from the last PR) The unhovered render blue and the layout effect blue are an identical sky blue

Happy to approve this as is even though it doesn't resolve #55 imo. We can always make another PR in the future to resolve that.

**Comment by kartikcho** [comment]
I've updated the colors again. Red is significantly darker now on hover. Also layout effects are purple but almost similar in contrast with unhovered render blues. 

I've changed the contrast of layout effects but if you're still having trouble, you can select a shade for it. 

Let me know if you can distinguish now @taneliang!

File: src/content-views/constants.js [hunk 1]
```diff
@@ -48,17 +48,17 @@ export const COLORS = Object.freeze({
...
   REACT_COMMIT: '#ff718e',
   REACT_COMMIT_SELECTED: '#FF5277',
-  REACT_COMMIT_HOVER: '#ff335f',
-  REACT_LAYOUT_EFFECTS: '#c49edd',
+  REACT_COMMIT_HOVER: '#ed0030',
+  REACT_LAYOUT_EFFECTS: '#c88ff0',
   REACT_LAYOUT_EFFECTS_SELECTED: '#934FC1',
   REACT_LAYOUT_EFFECTS_HOVER: '#601593',
-  REACT_PASSIVE_EFFECTS: '#c49edd',
+  REACT_PASSIVE_EFFECTS: '#d8aff4',
```
Comment (taneliang) [inline src/content-views/constants.js]: Was it intentional for passive and layout effect colors to only differ when they're not hovered over?

**Review by taneliang — APPROVED** [review]
Nice! I like the new reds, they're so much better.

File: src/content-views/constants.js (hunk 1 above)
Comment (kartikcho) [reply src/content-views/constants.js]: Ah, good catch. must've slipped through. No, they're supposed to be same.

**Pull request merged**